docs/detectors/uv.md

```diff
@@ -0,0 +1,6 @@
+# uv Detection
+## Requirements
+uv detection relies on a uv.lock file being present.
+
+## Detection strategy
```

Comment:
Could we place links here to `uv` official documentation?

Reply:
Links added.